README.md

```diff
@@ -1 +1,23 @@
-# helloMicroverse
\ No newline at end of file
+# Project Name:hello microverse project
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor
+browser like crome
+## Author
+ [Githab](https://github.com/settings/profile)
+[linkedin](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+[tiwtter](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
```

Comment:
Please change the generic platform names to your actual name/handle for that platform. It'll give you more visibility and make your documentation look more professional.

Additionally, follow the README template and include a header with your name before the links.

Reply:
@vechicin , please could you make clear what is the mistake here. this project is a previously accepted project and according to microverse rule a previously accepted project should be accepted.